client/tests/unit/example.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { shallowMount } from '@vue/test-utils'
+import HelloWorld from '@/components/HelloWorld.vue'
+
+describe('HelloWorld.vue', () => {
+  it('renders props.msg when passed', () => {
+    const msg = 'new message'
+    const wrapper = shallowMount(HelloWorld, {
+      propsData: { msg }
+    })
+    expect(wrapper.text()).toMatch(msg)
+  })
+})
```

Comment:
If you want working examples of more complex tests (using vuex, vuetify, vuelidate, routing, mocking, sinon), check out https://github.com/bcgov/lear/tree/master/coops-ui/tests/unit